realsense2_camera/scripts/mono_to_color.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import cv2
+from cv_bridge import CvBridge
+import rospy
+
+from sensor_msgs.msg import Image
+
+
+class MonoToColor(object):
+
+    def __init__(self):
+        rospy.init_node('mono_to_color')
+
+        mono_image_topic = rospy.get_param('~mono_image_topic')
+        color_image_topic = rospy.get_param('~color_image_topic')
+        self.frame_id = rospy.get_param('~frame_id', None)
+
+        self.bridge = CvBridge()
+
+        self.__color_image_pub = rospy.Publisher(color_image_topic,
+                                                 Image,
+                                                 queue_size=1)
+
+        self.__mono_image_sub = rospy.Subscriber(mono_image_topic,
+                                                 Image,
+                                                 self.__image_callback,
+                                                 queue_size=1)
+
+        rospy.loginfo('Initialized MonoToColor successfully.')
+
+    def __image_callback(self, mono_image_msg):
+        mono_image = self.bridge.imgmsg_to_cv2(mono_image_msg, 'passthrough')
```

Comment:
Use the normal `try` pattern here to avoid spewing out a bunch of stuff if a `SIGINT` is received here.